<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtpMonitorQTClass</class>
 <widget class="QMainWindow" name="CtpMonitorQTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CtpMonitorQT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.ico</normaloff>logo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>系统设置(&amp;F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
    <addaction name="action_D"/>
    <addaction name="action_T"/>
    <addaction name="action_M"/>
   </widget>
   <widget class="QMenu" name="menu_A">
    <property name="title">
     <string>关于(&amp;A)</string>
    </property>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_V"/>
   <addaction name="menu_A"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_C">
   <property name="text">
    <string>账户列表(&amp;C)</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>账号轧差视图(&amp;G)</string>
   </property>
  </action>
  <action name="action_D">
   <property name="text">
    <string>输出窗口(&amp;D)</string>
   </property>
  </action>
  <action name="action_T">
   <property name="text">
    <string>账户详细信息窗口(&amp;T)</string>
   </property>
  </action>
  <action name="action_M">
   <property name="text">
    <string>账户列表详细(&amp;M)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CtpMonitorQT.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>CtpMonitorQTClass</sender>
   <signal>destroyed()</signal>
   <receiver>CtpMonitorQTClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>709</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
